src/pages/specs/Dashboard.spec.js

```diff
@@ -0,0 +1,18 @@
+import { create } from 'react-test-renderer';
+import Dashboard from '../Dashboard';
+
+
+jest.mock('react-router-dom', () => ({ Link: 'Link' }));
+jest.mock('../styledComponents', () => ({ CardList: 'CardList', Error: 'Error', Loader: 'Loader' }));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
jest.mock("../styledComponents", () => ({ CardList: 'CardList', Error: 'Error', Loader: 'Loader' }));
```